Applications/FileManager/DirectoryView.cpp

```diff
@@ -108,8 +110,32 @@ void DirectoryView::set_view_mode(ViewMode mode)
     ASSERT_NOT_REACHED();
 }
 
+void print_path_history(int index, Vector<String> path_history)
```

Comment:
'path_history' should be passed by const-reference (const Vector<String>&) as this avoids making an unnecessary copy of the whole vector.